app/config/env.php

```diff
@@ -82,3 +82,20 @@
     $loader = new Loader\YamlFileLoader($container, new FileLocator(__DIR__ . '/cache_pool'));
     $loader->load($pool . '.yml');
 }
+
+// Yoochoose settings
+if ($value = getenv('YOOCHOOSE_CUSTOMER_ID')) {
+    $container->setParameter('ez_recommendation.default.yoochoose.customer_id', $value);
+}
+
+if ($value = getenv('YOOCHOOSE_LICENSE_KEY')) {
+    $container->setParameter('ez_recommendation.default.yoochoose.license_key', $value);
+}
+
+if ($value = getenv('YOOCHOOSE_SERVER_URI')) {
```

Comment:
In retrospective naming is somewhat misleading here as it's not really yochoose server uri, it's the one for eZ Publish install. ref: https://github.com/ezsystems/EzSystemsRecommendationBundle/blob/master/DependencyInjection/Configuration.php#L48